inc/spbc-settings.php

```diff
@@ -3065,6 +3072,83 @@ function spbc_field_scanner()
     echo '</div>';
 }
 
+add_action('wp_ajax_spbc_analysyis_files_stats__get_html', 'spbc__analysyis_files_stats__get_html');
+/**
+ * Retrieves HTML code block to layout files counters stats in the analysis accordion.
+ * @return string
+ */
+function spbc__analysyis_files_stats__get_html()
```

Comment:
Need to check a nonce in the start of the method.

Reply:
fixed https://github.com/CleanTalk/security-malware-firewall/pull/380/commits/fd73ddc248185d5e4b83a74cc9ba32251569eb88